modules/cluster-logging-clo-status.adoc

```diff
@@ -114,10 +112,7 @@ visualization:  <4>
 [id="cluster-logging-clo-status-message_{context}"]
 == Example condition messages
```

Comment:
Probably out of scope, but this seems like a new REF module

Reply:
Yeah, I'd leave this for a follow up I think for now.